index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>background-test</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+<main id="container">
+	<div class="nav-header">
+		<div id="first"><a href="#"><img src= "img/Icon-Mac.svg.png" alt="apple logo" id="img-nav"></a></div>
+		<div><a href="#">Store</a></div>
+		<div><a href="#">Mac</a></div>
+		<div><a href="#">iPod</a></div>
+		<div><a href="#">iPhone</a></div>
+		<div><a href="#"> iTunes</a></div>
+		<div><a href="#"> Support</a></div>
+		<div id="last">
+			<img src="img/magnifying-glass.png" alt="search icon" width="15" height="14" id="search-icon">
+			<input type="text" name="search" id="input" >
+		</div>
+	</div>
+
+    <section class="hero">
+		<a href="">
+			<h3>What will your Verse be?</h3>
+			<p>Everyone has something to share. See how people are using iPad in their unique ways to add to the world story.</p>
+			<p id="verse">Watch the "your Verse" film and explore the stories </p>
+		</a>
```

Comment:
The main title should be an `<h1>` tag for a better SEO and it is better to place an `<a>` inside each of the three hero elements (h3 and the two P tags)